mx_nuget.package/MendixBinaries.nuspec

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+  <metadata>
+    <id>Mendix.LibGit2Sharp.NativeBinaries</id>
+    <version>0.1</version>
+    <authors>LibGit2Sharp contributors</authors>
+    <owners>mendix</owners>
+    <licenseUrl>https://raw.githubusercontent.com/libgit2/libgit2/master/COPYING</licenseUrl>
+    <projectUrl>https://github.com/libgit2/libgit2sharp.nativebinaries</projectUrl>
+    <repository url="https://github.com/mendix/libgit2sharp.nativebinaries.git" type="git"/>
+    <iconUrl>https://raw.githubusercontent.com/libgit2/libgit2sharp/master/square-logo.png</iconUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>Native binaries for LibGit2Sharp</description>
+  </metadata>
+  <files>
+	<file src="buildMultiTargeting/**/*.*" />
```

Comment:
This file doesn't exist now; you can copy it and change it to specify NuGet package name inside.

Reply:
Done

Reply:
removed reference in MendixBinaries.nuspec and copied file to mx_nuget.package\buildMultiTargeting folder